docs/source/whatsnew.rst

```diff
@@ -1,21 +1,74 @@
 What's new in Hyperledger Fabric v2.x
 =====================================
 
+What's New in Hyperledger Fabric v2.3
+-------------------------------------
+
+Hyperledger Fabric v2.3 introduces two new features for improved orderer and peer operations.
+
+Orderer channel management without a system channel
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+To simplify the channel creation process and enhance the privacy and scalability of channels,
+it is now possible to create application channels without first creating a “system channel” managed by the ordering service.
+This process allows ordering nodes to join (or leave) any number of channels as needed, similar to how peers can participate in multiple channels.
+
+Benefits of the new process:
+
+* **Increased privacy:** Because all ordering nodes used to be joined to the system channel,
+  every ordering node in a network knew about the existence of every channel on that ordering service.
+  Now, an ordering node only knows about the channels it is joined to.
+* **Scalability:** When there is a large number of ordering nodes and channels defined on the system channel,
+  it can take a long time for ordering nodes to reach consensus on the membership of all the channels.
+  Now, an ordering service can scale horizontally in a decentralized fashion by independently joining ordering nodes to specific channels.
+* **Operational benefits**
+   * Simple process to join an ordering node to a channel.
+   * Ability to list the channels that the ordering node is a consenter on.
+   * Simple process to remove a channel from an ordering node, which automatically cleans up the blocks associated with that channel.
+   * Peer organizations do not need to coordinate with an admin of the system channel to create or update its MSP.
+
+For more information, see the :doc:`create_channel/create_channel_participation` topic.
+
+Ledger snapshot
+^^^^^^^^^^^^^^^
+
+It is now possible to take a snapshot of a peer's channel information, including its state database,
+and join new peers (in the same organization or different organizations) to the channel based on the snapshot.
+
+Using ledger snapshots has the following advantages:
+
+* **Peers don't need to process all blocks since genesis block:** Peers can join a channel without processing all
+  previous blocks since the genesis block, greatly reducing the time it takes to join a peer to an existing channel.
+* **Peers can join channels using latest channel configuration:** Because snapshots include the latest channel configuration,
+  peers can now join a channel using the latest channel configuration.
+  This is especially important if critical channel configuration such as orderer endpoints or MSP CA certificates have been updated since the genesis block.
```

Comment:
A nit, but I'd say it's more critical if the TLS CA certificates have changed.  In general, different signing certs aren't troublesome.

Reply:
ah right, done.